test/error.js

```diff
@@ -133,6 +133,15 @@ if (process.platform !== 'win32') {
 		t.is(signal, 'SIGINT');
 	});
 
+	test('error.signalDescription is defined', async t => {
+		const cp = execa('noop');
```

Comment:
Let's try to use `subprocess` instead of `cp` everywhere. Maybe we should add it to https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prevent-abbreviations.md

Reply:
Done in 1a5d1cb421129a60b2dd61d809704bc906fcac1d

I also opened a PR #379 to fix it in all other tests.